### ISSUE TITLE
Display build and deps error differently

### ISSUE THREAD

**Issue by lefessan** [OCamlPro/opam-builder#5]
Currently, a package that fails to install because it could not be compiled, or one of its deps failed to compile, are displayed in the same color. We should change that.

**Comment by gasche** [comment]
Indeed; I think that dependencies failure is actually a rather normal thing (if I depend on X and X turns out to be incompatible with an OCaml version, I don't want to have to update my own metadata to mark myself incompatible as well, it should not be counted as an error), so they should not be in red. White, or a light gray, would be best.
